usr/toolchain-nxdk.cmake

```diff
@@ -0,0 +1,29 @@
+set(NXDK_DIR $ENV{NXDK_DIR})
+
+set(CMAKE_SYSTEM_NAME Generic)
+set(TOOLCHAIN_PREFIX nxdk)
+
+set(WIN32 1)
+
+#FIXME: Include stuff for MinGW / Windows-GNU / Windows-Clang?
+#FIXME: How to set the ABI?
+
+set(NXDK_LINK_XBE "${NXDK_DIR}/usr/bin/${TOOLCHAIN_PREFIX}-exe2xbe -OUT:<TARGET>.xbe <TARGET>.exe")
+
+# cross compiler to use for C
+set(CMAKE_C_COMPILER "${NXDK_DIR}/usr/bin/${TOOLCHAIN_PREFIX}-cc")
+set(CMAKE_C_STANDARD_LIBRARIES "${NXDK_DIR}/lib/libwinapi.lib ${NXDK_DIR}/lib/xboxkrnl/libxboxkrnl.lib ${NXDK_DIR}/lib/libxboxrt.lib  ${NXDK_DIR}/lib/libpdclib.lib ${NXDK_DIR}/lib/libnxdk_hal.lib ${NXDK_DIR}/lib/libnxdk.lib ${NXDK_DIR}/lib/libnxdk_usb.lib") #"${CMAKE_CXX_STANDARD_LIBRARIES_INIT}"
+set(CMAKE_C_LINK_EXECUTABLE "${NXDK_DIR}/usr/bin/${TOOLCHAIN_PREFIX}-ld <FLAGS> <CMAKE_C_LINK_FLAGS> <LINK_FLAGS> <OBJECTS> -out:<TARGET>.exe <LINK_LIBRARIES>; ${NXDK_LINK_XBE}")
+
+# cross compiler to use for C++
+set(CMAKE_CXX_COMPILER "${NXDK_DIR}/usr/bin/${TOOLCHAIN_PREFIX}-c++")
+set(CMAKE_CXX_STANDARD_LIBRARIES "${CMAKE_C_STANDARD_LIBRARIES}")
+set(CMAKE_CXX_LINK_EXECUTABLE "${NXDK_DIR}/usr/bin/${TOOLCHAIN_PREFIX}-ld <FLAGS> <CMAKE_CXX_LINK_FLAGS> <LINK_FLAGS> <OBJECTS> -out:<TARGET>.exe <LINK_LIBRARIES>; ${NXDK_LINK_XBE}")
+
+
+function(create_xiso output_file input_folder)
```

Comment:
This does not work as intended, because dependencies are bad. Ideally we'd also be able to list files, rather than just doing it per folder